src/common/interfaces/domain-api-options.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface DomainApiOptions {
```

Comment:
I see this in another part of the PR. Is it a duplicate interface that we could consolidate / DRY up?

Reply:
🤦‍♂️ my mistake, sorry about that!

PR looks good to me.